guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -8,13 +8,11 @@ You can use a configuration management tool to configure multiple hosts at once.
 * The host must be using a supported operating system.
 For more information, see xref:supported-clients-in-registration_{context}[].
 ifdef::satellite,orcharhino,katello[]
-* {ProjectServer} and any {SmartProxyServers} must have a time-synchronization tool enabled and running and be synchronized with the same NTP server.
-For more information, see the following resources:
-** link:{InstallingServerDocURL}synchronizing-the-system-clock-with-chronyd_{project-context}[Synchronizing the system clock with chronyd] in _{InstallingServerDocTitle}_
+* The system clock on your {ProjectServer} and any {SmartProxyServers} must be synchronized across the network.
+If the system clock is not synchronized, SSL certificate verification might fail.
 ifdef::satellite[]
```

Comment:
I suggest to show the example (line 14) for all build flavors. The "if Satellite" macro is only needed if there are links to RH/RHEL docs.

Reply:
Done, thanks!